pkg/installmanager/dnscleanup.go

```diff
@@ -69,6 +72,27 @@ func cleanupAWSDNSZone(dnsZone *hivev1.DNSZone, region string, logger log.FieldL
 	return nil
 }
 
+// cleanupAzureDNSZone will return a DNS zone to the minimum set of DNS records
+func cleanupAzureDNSZone(dnsZone *hivev1.DNSZone, logger log.FieldLogger) error {
+	zoneLogger := logger.WithField("dnsZoneID", dnsZone.Spec.Zone)
+	zoneLogger.Info("cleaning up DNSZone")
+
+	creds, err := azureutils.GetCreds("")
+	if err != nil {
+		logger.WithError(err).Error("failed to get Azure creds")
```

Comment:
```suggestion
		zoneLogger.WithError(err).Error("failed to get Azure creds")
```
Although, I would prefer just overwriting the `logger` variable instead of having a `zoneLogger` variable.